### ISSUE TITLE
 Install ORB-SLAM2

### ISSUE THREAD

**Issue by Sanerok** [KaiChen1008/Sim-to-Real-Virtual-Guidance-for-Robot-Navigation#10]
Hi, in the step 3.6. Install ORB-SLAM2
You write that we need to download YOUR modification of ORB-SLAM2 which can restore maps, publish poses and construct both 2D and 3D maps simultaneously. But there is no link for it.
Can you share it?

**Comment by dxdpmdx** [comment]
Sim-to-Real-Virtual-Guidance-for-Robot-Navigation/localization_module/modified_ORB_SLAM2
it's already in the project
